scripts/operator-ce.yaml

```diff
@@ -0,0 +1,725 @@
+apiVersion: v1
```

Comment:
A couple of questions regarding this yaml:
1. What does the "-ce" suffix in the file name mean?
2. As we're deploying OLM in our cluster, isn't it possible to deploy this operator using OLM, creating the proper subscription? Is it this one https://artifacthub.io/packages/olm/community-operators/litmuschaos ? The yaml for the OLM subscruption (alpha channel) can be found here: https://operatorhub.io/install/alpha/litmuschaos.yaml

Reply:
ce is the meaning of chaos engine 
and i tried with the olm it didnt work there is some resources that are missing